src/TimeGridHeader.js

```diff
@@ -144,7 +148,12 @@ class TimeGridHeader extends React.Component {
             {resource && (
               <div className="rbc-row rbc-row-resource">
                 <div key={`resource_${idx}`} className="rbc-header">
```

Comment:
err looks liek there was an existing bug here, can you move the `key` to the correct dive (the one right above this one)